target/linux/ath79/dts/qca9533.dtsi

```diff
@@ -0,0 +1,222 @@
+
+				#size-cells = <0>;
+
+				pinctrl-single,bit-per-mux;
+				pinctrl-single,register-width = <32>;
+				pinctrl-single,function-mask = <0x1>;
+				#pinctrl-cells = <2>;
+
+				jtag_disable_pins: pinmux_jtag_disable_pins {
+					pinctrl-single,bits = <0x40 0x2  0x2>;
+				};
+			};
+
+			pll: pll-controller@18050000 {
+				compatible = "qca,qca9530-pll", "syscon";
+				reg = <0x18050000 0x48>;
+
+				#clock-cells = <1>;
+				clocks = <&ref>;
+				clock-names = "ref";
+				clock-output-names = "cpu", "ddr", "ahb";
+			};
+
+			wdt: wdt@18060008 {
+				compatible = "qca,ar9340-wdt", "qca,ar7130-wdt";
+				reg = <0x18060008 0x8>;
+
+				interrupts = <4>;
+
+				clocks = <&pll ATH79_CLK_AHB>;
+				clock-names = "wdt";
+			};
+
+			rst: reset-controller@1806001c {
+				compatible = "qca,ar7240-reset",
+						"qca,ar7100-reset";
+				reg = <0x1806001c 0xac>;
+
+				#reset-cells = <1>;
+			};
+
+			pcie0: pcie-controller@180c0000 {
+				compatible = "qcom,ar7240-pci";
+				#address-cells = <3>;
+				#size-cells = <2>;
+				bus-range = <0x0 0x0>;
+				reg = <0x180c0000 0x1000>, /* CRP */
+				      <0x180f0000 0x100>,  /* CTRL */
+				      <0x14000000 0x1000>; /* CFG */
+				reg-names = "crp_base", "ctrl_base", "cfg_base";
+				ranges = <0x2000000 0 0x10000000 0x10000000 0 0x04000000	/* pci memory */
+					  0x1000000 0 0x00000000 0x0000000 0 0x000001>;		/* io space */
+				interrupt-parent = <&cpuintc>;
+				interrupts = <0>;
+
+				interrupt-controller;
+				#interrupt-cells = <1>;
+
+				interrupt-map-mask = <0 0 0 1>;
+				interrupt-map = <0 0 0 0 &pcie0 0>;
+				status = "disabled";
+			};
+
+			usb0: usb@1b000000 {
+				compatible = "generic-ehci";
+				reg = <0x1b000000 0x1000>;
+
+				interrupts = <3>;
+				resets = <&rst 5>;
+				reset-names = "usb-host";
+				dr_mode = "host";
+
+				has-transaction-translator;
+				caps-offset = <0x100>;
+
+				phy-names = "usb-phy";
+				phys = <&usb_phy>;
+
+				status = "disabled";
+			};
+		};
+
+			wifi: wifi@180c0000 {
+				compatible = "qca,qca9530-wmac";
+				reg = <0x180c0000 0x30000>;
+
+				interrupt-parent = <&cpuintc>;
+				interrupts = <2>;
+
+				clock-names = "ref";
+				clocks = <&extosc>;
+
+//				nvmem-cell-names = "eeprom", "address";
+//				nvmem-cells = <&wmac_eeprom>, <&wmac_address>;
+
+				resets = <&rst 22>;
+			};
+
+		spi: spi@1f000000 {
+			compatible = "qca,ar9530-spi", "qca,ar7100-spi";
+			reg = <0x1f000000 0x10>;
+
+			clocks = <&pll ATH79_CLK_AHB>;
+			clock-names = "ahb";
+
+			status = "disabled";
+
+			#address-cells = <1>;
+			#size-cells = <0>;
+		};
+	};
+
+	usb_phy: usb-phy@18030000 {
+		compatible = "qca,ar7200-usb-phy";
+		reg = <0x18030000 0x100>;
+		#phy-cells = <0>;
+
+		reset-names = "usb-phy", "usb-suspend-override";
+		resets = <&rst 4>, <&rst 3>;
+
+		status = "disabled";
+	};
+
+};
+
+&mdio0 {
+	resets = <&rst 22>;
+	reset-names = "mdio";
+};
+
+&eth0 {
+	compatible = "qca,qca9530-eth", "syscon";
+	pll-data = <0x82000101 0x80000101 0x80001313>;
```

Comment:
If you want to set pll-data you need also pll-reg and pll-handle.
As i know for qca9530-eth `set_pll` isn´t called for now too in `ag71xx_main.c`


...